tests/test_website_monitor.py

```
@@ -0,0 +1,1032 @@
+    def test_get_realistic_user_agent_varies_versions(self, website_monitor):
+        """
+        Positive test: User agent uses randomized Chrome versions.
+
+        Objective: Multiple calls may return different versions.
+        """
+        # Arrange & Act
+        versions = set()
+        for _ in range(20):
+            ua = website_monitor._get_realistic_user_agent()
+            # Extract version number
+            import re
```

Ruff (0.14.13)

Redefinition of unused `re` from line 19: `re` redefined here

Remove definition: `re`

(F811)